src/audio/module_adapter/module/iadk_modules.c

```diff
@@ -228,6 +284,13 @@ static int iadk_modules_set_configuration(struct processing_module *mod, uint32_
 					  size_t fragment_size, uint8_t *response,
 					  size_t response_size)
 {
+	if (mod->is_native_sof) {
+		struct module_interface *mod_in =
+					(struct module_interface *)mod->priv.module_adapter;
+
+		return mod_in->set_configuration(mod, config_id, pos, data_offset_size, fragment,
```

Comment:
why no else here for iadk call ?

Reply:
@lgirdwood because in above cases there's a single `return` statement in the end and they only assign `ret` on `if` branches, whereas here there's a `return` directly under the `if` so you don't need an `else`. Of 3 variants:
1.
```
if (a)
	ret = f1();
else
	ret = f2();
return ret;
```
2.
```
if (a)
	return f1();
else
	return f2();
```
3.
```
if (a)
	return f1();
return f2();
```
we try to avoid (2) (and I think checkpatch complains about it), both (1) and (3) are acceptable by the style, personally I prefer (3).

Reply:
@lyakh I mean in reference to some ops calling both native and iadk apis and some ops calling either iadk or native api. i.e. we need som comments to explain.

Reply:
pls ignore, these are now visible in the new diff and I have coffee.